src/themes/elife/styles.css

```diff
@@ -123,6 +125,17 @@ main {
   }
 }
 
+:--datePublished {
```

Comment:
Better to just target the date in the content header, rather than make it global? At the moment looks like there's quite a bit of resetting of this on [line 173 & following](https://github.com/stencila/thema/pull/142/files#diff-44d5a42c7d7f66640339dcf314193b4cR173-R180).